src/PushNotifications.php

```diff
@@ -60,7 +64,53 @@ public function __construct($options, $client = null) {
     }
   }
 
+  private function makeRequest($method, $path, $pathParams, $body = null) {
+    $escapedPathParams = [];
+    foreach($pathParams as $k => $v) {
```

Comment:
Do we want to have spaces after the keyword `foreach`?

It would be consistent with `if`.